js/main.js

```diff
@@ -0,0 +1,57 @@
+//sets endorsement section animation based on scrollbar position
+var blockquote = document.getElementById('blockquote');
+var quoteHeight = document.getElementById('quote').scrollHeight;
+var animateQuote = function() {
+  blockquote.classList.add("endorsement_quote_animated");
+};
+var evalHeight = function() {
+  if (window.scrollY*4 >= quoteHeight) {animateQuote();}
+};
+document.addEventListener("scroll", evalHeight, false);
+
+//password form validation
+  //http://www.the-art-of-web.com/javascript/validate-password/
+var pswdOne = document.getElementById('password');
+var pswdTwo = document.getElementById('confirm');
+var submit = document.getElementById('subscribe_button');
+function checkForm() {
+  if(pswdOne.value != "" && pswdOne.value == pswdTwo.value) {
+    if(pswdOne.value.length < 6) {
+      pswdTwo.setCustomValidity("Error: Password must contain at least six characters!");
+      pswdOne.focus();
+      return false;
+    } else {
+      pswdTwo.setCustomValidity('');
+    }
+    re = /[0-9]/;
+    if(!re.test(pswdOne.value)) {
```

Comment:
I could just be missing it, but do you `var` declare for `re` anywhere? if not, it is now polluting the global namespace.